test.bib

```diff
@@ -0,0 +1,12 @@
+@article{dummy,
```

Comment:
Please use `src/main/resources/Chocolate.bib` instead of putting a .bib file in the root of the repository.